pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -280,6 +280,13 @@ spec:
               postStart:
                  exec:
                     command: ["/bin/sh", "-c", "sleep 2"]
+              # Removes the sock file that is used for communication between
+              # cstor-pool and cstor-pool-mgmt containers, so that, cstor-pool-mgmt
+              # of new pool pod instance will NOT communicate with cstor-pool
+              # container of terminating pod instance.
+              preStop:
+                 exec:
+                    command: ["/bin/bash", "-c", "rm /tmp/uzfs.sock"]
```

Comment:
Why do we share the tmp folder from the node in case of cstor pools instead of expty dir as we do in target deployment.